Api/Problem.hpp

```diff
@@ -361,7 +361,7 @@ class Problem
     friend class Problem;
     void validate() const;
 
-    void prepareOptionsReader(OptionsReader& rdr);
+    // void prepareOptionsReader(OptionsReader& rdr);
```

Comment:
In Api, this looks pretty inconsequential, but why comment it out?

Reply:
I think we did already talk about that, but in order to have i there publicly as well:
That was dead code, and instead of deleting it straight away i just commented it out.